tests/worker/bundle_state_test.py

```diff
@@ -33,17 +33,66 @@ def test_bundle_info_serialization(self):
             ],
             'location': 'location',
         }
+
+    def tearDown(self) -> None:
+        del self.bundle_info_fields
+
+    def test_bundle_info_serialization(self):
```

Comment:
When I said adding a test, I meant not testing the serialization (which is not the main part of the logic), but rather the `cl` command in `test_cli.py` that has references the same bundle twice.  Basically a minimal test case that didn't work before but works now.

Reply:
Oh, I see. I originally thought since we have altered the `dependencies` logic, it would be helpful to add tests to verify that.  But I agree that testing on `cl` command is more important. I have added two test cases in `test_cli.py`.